package.json

```diff
@@ -43,7 +43,11 @@
     "testUtils",
     "webExtensionApis/study/api.js",
     "webExtensionApis/study/schema.json",
-    "webExtensionApis/study/src/telemetry.js"
+    "webExtensionApis/study/src/telemetry.js",
```

Comment:
I am not loving "weeUtils" as a name, but I don't have one to suggest.  I do like the idea of stuff to help out authors.  I am not sure that this is the best way to document and create that. 

+0

Reply:
Moved discussion to https://github.com/mozilla/shield-studies-addon-utils/issues/213